queries.sql

```diff
@@ -25,3 +25,76 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 /* QUERY TABLE FOR ANIMALS WITH WEIGHT BETWEEN 10.4KG AND 17.3KG */
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+
+/** DAY 2 ***/
+
+/*** TRANSACTIONS ***/
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+select * from animals;
+COMMIT;
+select * from animals;
+
+-- update other species to pokemon
+BEGIN;
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+UPDATE 5
+select * from animals;
+COMMIT;
+select * from animals;
+
+
+-- delete all animals
+BEGIN;
+DELETE FROM animals;
+DELETE 11
+select * from animals;
+ROllBACK;
+
+--delete animals born after 2022 and update weight_kg
+
+begin;
+BEGIN
+delete from animals where date_of_birth > DATE '2022-01-01';
+DELETE 1
+
+--savepoint
+SAVEPOINT sp1;
+SAVEPOINT
+
+--update all weight_kg
+update animals set weight_kg = weight_kg * -1;
+UPDATE 10
+select * from animals;
+
+--roll back to sp1
+ROLLBACK TO sp1;
+ROLLBACK
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+UPDATE 4
+select * from animals;
+commit;
+
+--how many animals are ther 
+select count(*) from animals;
+
+--how many animal has tried to escape 
+select count(*) from animals where escape_attempt = 0;
+
+-- what is the average weight of animals?
+select avg(weight_kg) from animals;
+
+-- who escaped the most, neutered or not neutered?
+select neutered, sum(escape_attempt) from animals Group by neutered;
+
+--what is the minimum weight of each type of animal ?
+select species, min(weight_kg) from animals group by species;
+
```

Comment:
- [ ] You see the way you label your query code here with the query questions. I suggest you do the same to your screenshots in the pull request.